server/protocols/webadmin/webadmin-mailbox/pom.xml

```diff
@@ -156,5 +156,38 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>swagger-json</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.github.kongchen</groupId>
+                        <artifactId>swagger-maven-plugin</artifactId>
+                        <configuration>
+                            <apiSources>
```

Comment:
you should inherit configuration from parent module instead of repeating yourself

Reply:
The resulting file `swaggerFileName` name is not the same.